src/formElementPlugins/checkbox/src/ApplicationView.tsx

```diff
@@ -44,6 +47,16 @@ export const CheckboxDisplay: React.FC<{
             toggle={type === 'toggle'}
             slider={type === 'slider'}
           />
+          {cb.key === 'other' && (
```

Comment:
> This can be manually broken by assigning a key with 'other' as a value.

Hmm, not 100% sure what you mean, but I'll have a play and see if I can reproduce what you're seeing.

Reply:
You can configure a checkbox option with a simple string as one of the array items, or you can configure it with an object like:

{
 "key": 0,
 "label": "Option 1",
 "selected": true
}

And if the key is set as 'other', it will duplicate the other open text box like above.

Bit of a niche case.

Reply:
Yeah, so "other" is essentially a "reserved keyword" that you can't use as a checkbox key (can't really see why you'd ever need to -- most of the time we're not even using keys, just leaving it as default `0`, `1`, `2`, etc.)